include/deal.II/matrix_free/shape_info.templates.h

```diff
@@ -170,6 +171,64 @@ namespace internal
         }
     }
 
+    // TODO. This should be implemented into
+    // get_element_type_specific_information() and in the FE and not stand alone
+    // here.
+    template <int dim>
+    std::vector<unsigned int>
+    get_lexicographic_numbering_raviart_thomas(const unsigned int degree,
+                                               const unsigned int n_dofs_face)
+    {
```

Comment:
Question: Is this numbering the same as https://github.com/dealii/dealii/blob/0d07d611007fdbed3ca937c22974b7b1813cfa70/source/fe/fe_raviart_thomas_nodal.cc#L152-L207 If yes, I would suggest to move the code to `include/deal.II/fe/fe_raviart_thomas_nodal.h` and into an `internal` namespace there, so we avoid writing the same code twice.